src/ntttcp-for-linux-log.json

```diff
@@ -0,0 +1,110 @@
+{
```

Comment:
Generally we don't need to check in the sample log file. Please remove it.

Reply:
Has been removed.